README.md

```diff
@@ -91,6 +91,23 @@ Sometime you want to obtain some random bytes (hexadecimal). For example, you wa
 $randomRGB = Randomize::byte()->length(3)->generate();
 ```
 
+### Generate a Date
+Sometimes you want to obtain a random date (default min - max range of First day of current year - Last day of current year). For example, you want to generate a random date of birth:
```

Comment:
For example, you want to generate a random date:

Reply:
Got it. I'll change that